integ/test_init/integ_test/README.md

```diff
@@ -0,0 +1,58 @@
+
+# Welcome to your CDK Python project!
+
+This is a blank project for CDK development with Python.
```

Comment:
is it a blank project? I think it does something...

Reply:
This is auto created by CDK, I will modify some incorrect content in it..